### DISCUSSION TITLE
Axis label bounds

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#1685] [General]
I'm trying to position the x/y axis labels away from the edge of a chart when specifying a margin and frame colour.

However, I can only reposition the labels for each axis in one direction via the `labelOffset` property. In the other direction, they're positioned right up against the edge of the chart.

Here's a live demo of the issue: https://observablehq.com/d/9534efd1622cda76

![image](https://github.com/observablehq/plot/assets/1482075/5ab6a5f1-2dac-4d33-8d20-703af6fd5437)

**Comment by Fil** [comment]
There is an open issue for this bug https://github.com/observablehq/plot/issues/1414; please upvote!

**Reply by dgwyer** [reply]
Thanks, done!

**Comment by mbostock** [comment]
~~You can use the **dx** and **dy** options to move things around, too.~~ Oops, you can’t: this doesn’t move the axis label. Also consider using a plain text mark if you just want to render text (instead of an axis mark with empty ticks).

**Reply by dgwyer** [reply]
I do want to use the axis mark (I just set empty ticks for the demo), but using `dx`, and `dy` only seem to affect the tick marks and not the axis label.

I've added a second chart in the notebook to show this. Is my implementation wrong or is it that I just need to wait for #1414 to be merged?

https://observablehq.com/d/9534efd1622cda76

**Reply by mbostock** [reply]
Oh, right, sorry I forgot about that…

**Reply by dgwyer** [reply]
No problem. :)
